### DISCUSSION TITLE
Support for acceleration-based policies?

### DISCUSSION THREAD

**Discussion by benjamin-macadam** [google/brax#309] [Q&A]
I was wondering if you have any plans to support training acceleration-based policies (I have a few that I would like to try out with Brax), or if there is any workaround to using joint accelerations in Brax/computing joint forces from joint accelerations.

**Comment by btaba** [comment]
Hi @benjamin-macadam thanks for using brax! Could you clarify what you mean by acceleration-based policies? You can supply a `tau`, a joint actuation force to the engine, [example](https://github.com/google/brax/blob/main/brax/v2/envs/ant.py#L248), [xml ref](https://github.com/google/brax/blob/main/brax/v2/envs/assets/ant.xml#L83-L92).  Is that what you mean or something else?
